### PR TITLE
Changelog for 0.18.0

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+## [0.18.0](https://github.com/redhat-developer/vscode-xml/milestone/22?closed=1) (9 August, 2021)
+
+### Enhancements
+
+ * Added CodeLens that displays referenced grammars at the top of an XML file. See [#569](https://github.com/redhat-developer/vscode-xml/pull/569).
+ * Added CodeAction to bind an XML document to an existing schema. See [#515](https://github.com/redhat-developer/vscode-xml/issues/515).
+ * Document lemminx debug process with vscode-xml. See [#543](https://github.com/redhat-developer/vscode-xml/issues/543).
+ * If the workspace is not trusted, "Resolve external entities" is set to false. See [#537](https://github.com/redhat-developer/vscode-xml/issues/537).
+ * Command to bind an XML document to an existing schema. See [#514](https://github.com/redhat-developer/vscode-xml/issues/514).
+ * Add documentation for binding to grammar/schema. See [#567](https://github.com/redhat-developer/vscode-xml/issues/567).
```
Comment (AlexXuChen) [inline CHANGELOG.md:12]: Change "Add" to "Added"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+ * Add documentation for `Bind to grammar/schema file` command. See [#561](https://github.com/redhat-developer/vscode-xml/issues/561).
+ * Add documentation for CodeLens support in features docs. See [#562](https://github.com/redhat-developer/vscode-xml/issues/562).
+ * Add documentation for formatting in features docs. See [#560](https://github.com/redhat-developer/vscode-xml/issues/560).
+ * Update documentation for `closingBracketNewLine` when `splitAttributesIndentSize` is enabled. See [#550](https://github.com/redhat-developer/vscode-xml/pull/550).
+ * Added debug instructions for debugging lemminx with vscode-xml. See [#549](https://github.com/redhat-developer/vscode-xml/pull/549).
+ * Add documentation for workspace trust interaction with `resolveExternalEntities`. See [#546](https://github.com/redhat-developer/vscode-xml/pull/546).
+ * Collect telemetry data about text documents that are opened: file extension, grammar binding strategy, grammar resolving strategy. See [eclipse/lemminx#1066](https://github.com/eclipse/lemminx/issues/1066)
```
Comment (datho7561) [inline CHANGELOG.md:19]: forgot period at end of list item

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+
+### Enhancements
+
+ * Added CodeLens that displays referenced grammars at the top of an XML file. See [#569](https://github.com/redhat-developer/vscode-xml/pull/569).
+ * Added CodeAction to bind an XML document to an existing schema. See [#515](https://github.com/redhat-developer/vscode-xml/issues/515).
+ * Document lemminx debug process with vscode-xml. See [#543](https://github.com/redhat-developer/vscode-xml/issues/543).
+ * If the workspace is not trusted, "Resolve external entities" is set to false. See [#537](https://github.com/redhat-developer/vscode-xml/issues/537).
+ * Command to bind an XML document to an existing schema. See [#514](https://github.com/redhat-developer/vscode-xml/issues/514).
+ * Add documentation for binding to grammar/schema. See [#567](https://github.com/redhat-developer/vscode-xml/issues/567).
+ * Add documentation for `Bind to grammar/schema file` command. See [#561](https://github.com/redhat-developer/vscode-xml/issues/561).
```
Comment (AlexXuChen) [inline CHANGELOG.md:13]: Same as above

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+ * Add documentation for CodeLens support in features docs. See [#562](https://github.com/redhat-developer/vscode-xml/issues/562).
```
Comment (AlexXuChen) [inline CHANGELOG.md:14]: Same as above

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+ * Add documentation for formatting in features docs. See [#560](https://github.com/redhat-developer/vscode-xml/issues/560).
```
Comment (AlexXuChen) [inline CHANGELOG.md:15]: Same as above

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,33 @@
...
+ * Update documentation for `closingBracketNewLine` when `splitAttributesIndentSize` is enabled. See [#550](https://github.com/redhat-developer/vscode-xml/pull/550).
+ * Added debug instructions for debugging lemminx with vscode-xml. See [#549](https://github.com/redhat-developer/vscode-xml/pull/549).
+ * Add documentation for workspace trust interaction with `resolveExternalEntities`. See [#546](https://github.com/redhat-developer/vscode-xml/pull/546).
```
Comment (AlexXuChen) [inline CHANGELOG.md:18]: Same as above

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+ * Collect telemetry data about text documents that are opened: file extension, grammar binding strategy, grammar resolving strategy. See [eclipse/lemminx#1066](https://github.com/eclipse/lemminx/issues/1066).
+
+### Bug Fixes
+
+ * Fix stackoverflow and resource leak when calculating folding ranges. See [#538](https://github.com/redhat-developer/vscode-xml/issues/538).
+ * Indent closing bracket as much as attributes when `splitAttributes` and `closingBracketNewLine` are enabled. See [#516](https://github.com/redhat-developer/vscode-xml/issues/516).
```
Comment (AlexXuChen) [inline CHANGELOG.md:24]: "Aligned tag closing bracket with attribute indent when..."

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,33 @@
...
+ * Add documentation for CodeLens support in features docs. See [#562](https://github.com/redhat-developer/vscode-xml/issues/562).
+ * Add documentation for formatting in features docs. See [#560](https://github.com/redhat-developer/vscode-xml/issues/560).
+ * Update documentation for `closingBracketNewLine` when `splitAttributesIndentSize` is enabled. See [#550](https://github.com/redhat-developer/vscode-xml/pull/550).
+ * Added debug instructions for debugging lemminx with vscode-xml. See [#549](https://github.com/redhat-developer/vscode-xml/pull/549).
+ * Add documentation for workspace trust interaction with `resolveExternalEntities`. See [#546](https://github.com/redhat-developer/vscode-xml/pull/546).
+ * Collect telemetry data about text documents that are opened: file extension, grammar binding strategy, grammar resolving strategy. See [eclipse/lemminx#1066](https://github.com/eclipse/lemminx/issues/1066).
+
+### Bug Fixes
+
+ * Fix stackoverflow and resource leak when calculating folding ranges. See [#538](https://github.com/redhat-developer/vscode-xml/issues/538).
```
Comment (AlexXuChen) [inline CHANGELOG.md:23]: Change "Fix" to "Fixed"

**Review by AlexXuChen — CHANGES_REQUESTED** [review]

**Review by AlexXuChen — APPROVED** [review]
LGTM

**Comment by AlexXuChen** [comment]
Thanks @datho7561!

**Comment by datho7561** [comment]
Thanks for the review!

**Pull request merged**